WireAuthentication/Sources/WireAuthenticationUI/Views/Root/RootViewModel.swift

```diff
@@ -24,7 +24,16 @@ import WireAuthenticationAPI
 @MainActor
 public final class RootViewModel: ObservableObject, Router {
 
+    enum ActiveSheet: Identifiable, Hashable {
+        var id: Self { self }
+
+        case authFlow
+        case noHistory(userID: UUID, cookies: [HTTPCookie])
+    }
```

Comment:
suggestion: call this `Destination` or `SheetDestination` or `ModalDestination` and move it to `RootView`, to align with other destinations.

Reply:
done